wercker.yml

```diff
@@ -0,0 +1,10 @@
+box: centos:6
+build:
+  steps:
+    - script:
+        name: install rmagick
+        code: |
+          sudo yum install -y ruby ruby-devel rubygems ImageMagick-devel gcc
```

Comment:
So this just installs the default system ruby and tries to install against that? Were you able to get it to fail against the previous build that broke?